### ISSUE TITLE
Test Invenio Connector

### ISSUE THREAD

**Issue by matthewjmckinley** [omeka-s-modules/DataRepositoryConnector#4]
This is specifically for the Invenio Connector, since the other data repository connectors have already been tested.

These are the working Invenio repositories I was able to find. You can put this in the `Main Invenio Field`. To find a `Search Query`, visit the site and search for various terms (nouns like colors or countries tend to work well) until you find a reasonably sized record result to test with:

- http://opendata.cern.ch
- https://b2share.eudat.eu
- https://data.caltech.edu
- https://museumbollandianum.be
- https://www.openaccessrepository.it

**Testing Tasks**
- Test importing collections of various sizes, adjusting all configuration parameters and making sure they are reflected in the imported collection
- Test importing files into Omeka S. Note that many data file formats aren't on the list of supported media types/extensions in the Omeka S settings--this may cause an import error in which case you'll have to add the extensions/media types manually, or choose another collection
- You can try importing an entire Invenio instance by leaving the `Search Query` field blank, however the test instances given above are all quite large. Might be possible with the  which has around 11,000 items, but if it's just working even by lowering the `Limit` setting, don't worry about it. 
- Test undoing an import 

Finally, note that this connector is having to crosswalk from a lot of slightly different metadata configurations/schema to Dublin Core, so not ALL of the metadata fields will necessarily cross over--just take note of the major ones like Title, Creator, Description and make sure no fields are crosswalked to an unexpected/incorrect field.

Thanks!

**Comment by katknow** [comment]
This is working really well for me. Tried all example sites you provided and did all tasks. As you mentioned, the quality of metadata etc. is dependent on the individual site. If things get slow on big imports, I just bump the limit to a few hundred (higher if no files) and it works fine. Sometimes the description imports all the html tags into the field, but that also seems site dependent so I don't think it's on the Omeka end.

Only thing I noticed that needs to be fix is the help text under "limit" mentions Zenodo instead of Invenio:
![image](https://user-images.githubusercontent.com/84726696/208967705-57db5bf9-2da6-42c9-b0ba-446e8234b791.png)
